frontend/src/lib/integrations/googleAdsIntegrationLogic.ts

```diff
@@ -0,0 +1,42 @@
+import { actions, connect, kea, key, path, props } from 'kea'
+import { loaders } from 'kea-loaders'
+import api from 'lib/api'
+import { preflightLogic } from 'scenes/PreflightCheck/preflightLogic'
+
+import { GoogleAdsConversionActionType } from '~/types'
+
+import type { googleAdsIntegrationLogicType } from './googleAdsIntegrationLogicType'
+
+export const googleAdsIntegrationLogic = kea<googleAdsIntegrationLogicType>([
+    props({} as { id: number }),
+    key((props) => props.id),
+    path((key) => ['lib', 'integrations', 'googleAdsIntegrationLogic', key]),
+    connect({
+        values: [preflightLogic, ['siteUrlMisconfigured', 'preflight']],
```

Comment:
What are these used for?

Reply:
They are not being used - I've left them in after copying the slackIntegrationLogic.ts file https://github.com/PostHog/posthog/blob/5244e546d1e8224bda1f8f9658d578c82df90a23/frontend/src/lib/integrations/slackIntegrationLogic.ts#L14-L16

Removed them in [`fc252b3` (#26088)](https://github.com/PostHog/posthog/pull/26088/commits/fc252b396513061395db9985244381d13713e1ba)